packages/trader/src/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -204,21 +204,23 @@ class OpenPositions extends React.Component {
         row_obj.is_unsupported
             ? {
                   component: (
-                      <Localize
-                          i18n_default_text='This trade type is currently not supported on {{website_name}}. Please go to <0>Binary.com</0> for details.'
-                          values={{
-                              website_name,
-                          }}
-                          components={[
-                              <a
-                                  key={0}
-                                  className='link link--orange'
-                                  rel='noopener noreferrer'
-                                  target='_blank'
-                                  href={urlFor('user/portfoliows', undefined, undefined, true)}
-                              />,
-                          ]}
-                      />
+                      <div className='unsupported-trade-type'>
```

Comment:
Can we refactor this into a component? I see it is repeating in `profit-table.jsx` as well, we can pass in the arg for `urlFor` as a prop to that component.

Reply:
Indeed I was thinking about this. As it's used in just 2 places I didn't go for it but yea we can and  [I immediately regret my decision](https://www.reddit.com/r/DotA2/comments/gduivs/ballet_lesson/?utm_source=share&utm_medium=web2x)